### ISSUE TITLE
Not include Android dependencies

### ISSUE THREAD

**Issue by iadcialim** [mikepenz/AboutLibraries#667]
Question:
Is there a way to exclude dependencies that are related to Android?
Im thinking of not including them so that my list is not that too long and just focus on other 3rd parties other than Android.

**Comment by mikepenz** [comment]
Please have a look here: https://github.com/mikepenz/AboutLibraries#exclude-libraries

**Comment by iadcialim** [comment]
hi @mikepenz Thanks  fort his but
I tried 
```
aboutLibraries {
    exclusionPatterns = [~"androidx_.*"]
}

```
But this code
```
val libraries = Libs(this).libraries
for (lib in libraries) {
    Log.e("AboutLibraries", "classPath=${lib.libraryArtifactId} ${lib.libraryName} from ${lib.author}")
}
```

still returns the the libs from  AOSP like `androidx.activity:activity:1.2.3` 
Same for the LibsBuilder, the list still includes the excluded library.

**Comment by mikepenz** [comment]
@iadcialim thank you for the report. will be fixed in v8.9.3.
